### DISCUSSION TITLE
Can pretty-printing for quote/quasiquote/unquote be disabled for write or display?

### DISCUSSION THREAD

**Discussion by Zambito1** [leppie/IronScheme#133] [Discussions] [answered]
I'm looking to write expressions to a port which can be read and evaluated by a non-Scheme Lisp, and which does not use the same special syntax for quasiquote and unquote, but it uses the same quasiquote and unquote keywords.

Here is the behavior of Guile Scheme, which does what I would like:

```scheme
scheme@(guile-user)> (write (quote (quasiquote (foo (unquote x))))) (newline)
(quasiquote (foo (unquote x)))
scheme@(guile-user)> (write '`(foo ,x)) (newline)
(quasiquote (foo (unquote x)))
```

Contrast this with IronScheme:

```scheme
> (write (quote (quasiquote (foo (unquote x))))) (newline)
`(foo ,x)
> (write '`(foo ,x)) (newline)
`(foo ,x)
```

The output of Guile can be directly `(read)` by the other Lisp as I intend, but the special syntax used by `write` from IronScheme cannot be directly read. Is there a way to configure IronScheme to not use the special quasiquote and unquote characters for `write`?

**Comment by leppie** [comment]
I have transfer this to issues as a feature-request. See #135

**Comment by leppie** [comment, marked as answer]
https://github.com/IronScheme/IronScheme/releases/tag/1.0.307
